packages/stream_video/lib/src/call/call_events.dart

```diff
@@ -1037,6 +1037,28 @@ class StreamCallUserUnblockedEvent extends StreamCallEvent {
       ];
 }
 
+/// Event that is triggered when the user is kicked from a call.
+class StreamCallUserKickedEvent extends StreamCallEvent {
+  const StreamCallUserKickedEvent(
+    super.callCid, {
+    required this.createdAt,
+    required this.user,
+    required this.kickedByUser,
+  });
+
+  final DateTime createdAt;
+  final CallUser user;
+  final CallUser? kickedByUser;
+
+  @override
+  List<Object?> get props => [
+        ...super.props,
+        createdAt,
+        user,
+        kickedByUser,
+      ];
+}
+
```

Comment:
Should we also leave the call when we are kicked? Or do we get another call event that would trigger a leave?